app/src/main/java/com/github/dedis/student20_pop/model/event/Event.java

```diff
@@ -9,7 +9,7 @@
 import java.util.Objects;
 
 /**
- * Class modeling an Event
+ * Class modeling an abstract Event
```

Comment:
Yes, but the class itself isn't abstract (yet). It could be, as long as we change the types and use particular events in PoPApplication's dummyEvents. Doesn't have to be done in this PR but I think it should be done some time.

Reply:
Yes I wanted to change it to abstract, changed  the doc, faced too many problems, rollback but forgot the doc